catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswStore.java

```diff
@@ -219,7 +225,9 @@ public UpdateResponse update(UpdateRequest updateRequest) throws IngestException
                     .add(new UpdateAction(metacard, insertTypeName, null));
         }
         try {
+            LOGGER.trace("Sending {} transactions", transactionRequest.getInsertActions().size());
             TransactionResponseType response = csw.transaction(transactionRequest);
+            LOGGER.trace("Received response {}", response.toString());
```

Comment:
Could the response be quite large (and especially if its doing any xml marshalling it could be slow)? Just wondering if we want to wrap this in a if `LOGGER.isTraceEnabled` block. (though it would not look near as clean). If we do we should for all the ones doing response.toString()`

also is this formatted? i'd assume 228 would have been collapsed to 2 lines by our formatter.